projects/projects.json

```diff
@@ -2,12 +2,7 @@
     "projects": [
         {
             "name": "falco",
-            "organization": "falcosecurity",
-            "sub_components": [
-                "ebpf",
-                "modern-ebpf",
-                "kmod"
-            ]
+            "organization": "falcosecurity"
```

Comment:
Currently the deploy step fails because not all pods are ready due to node labels. 

Once the falco pods use the new `node-role.kubernetes.io/benchmark` label we can add the drivers again but the array name will change to `configs`.

```
Run kubectl apply -f manifest.yaml
gitrepository.source.toolkit.fluxcd.io/falco-cncf-green-review-testing created
kustomization.kustomize.toolkit.fluxcd.io/falco-cncf-green-review-testing created
timed out waiting for the condition on pods/falco-event-generator--driver-kmod-7b7c7cfc4f-rs7sz
timed out waiting for the condition on pods/falco-event-generator-driver-ebpf-785c6cc7dc-gr6j8
timed out waiting for the condition on pods/redis-driver-ebpf-5fd857f98-rg5j2
timed out waiting for the condition on pods/redis-driver-kmod-6c45747c64-6b9qs
timed out waiting for the condition on pods/stress-ng-driver-ebpf-5c88dff8b9-g9nz8
timed out waiting for the condition on pods/stress-ng-driver-kmod-59d6964b47-66wld
timed out waiting for the condition on pods/redis-driver-modern-ebpf-64cb4d8d78-6b5rr
timed out waiting for the condition on pods/falco-event-generator-driver-modern-ebpf-64674f78bf-xmjbf
timed out waiting for the condition on pods/stress-ng-driver-modern-ebpf-54685d8dff-5nqnk
timed out waiting for the condition on pods/falco-driver-modern-ebpf-fd77s
Error: Process completed with exit code 1.
```

https://github.com/cncf-tags/green-reviews-tooling/actions/runs/9757202145/job/26929084030